core/src/main/java/com/backblaze/b2/client/structures/B2ApplicationKey.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2018, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+
+package com.backblaze.b2.client.structures;
+
+import com.backblaze.b2.json.B2Json;
+
+import java.util.Objects;
+import java.util.TreeSet;
+
+/**
+ * Response from b2_create_key and b2_delete_key.  Included in response from b2_list_keys.
```

Comment:
"Response from b2_create_key".  i think that comment is a little out-of-date.  I think you're returning a B2CreatedApplicationKey from b2_create_key now.  :)